### PR TITLE
Updated generator

### PR DESCRIPTION
**Pull request by juan-blasco**
Updates list:
- Latest versions in POM
- Latest changes on docker maven plugin
- Fixes to Jenkins DSL jobs file templates
- Fix to application.yml data rest property
- Fix to application.yml connector allowed keys and secrets
- Remove README load testing section

### CODE REVIEW

File: generators/java/templates/pom.xml [hunk 1]
```diff
@@ -4,39 +4,41 @@
...
 	<artifactId><%= integrationName %>-connector</artifactId>
 	<name><%= integrationName %>-connector</name>
-	<version>0.1-SNAPSHOT</version>
+	<version>0.1.0-SNAPSHOT</version>
 
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>1.5.9.RELEASE</version>
+		<version>1.5.10.RELEASE</version>
```
Comment (nicolas-antomarioni) [inline generators/java/templates/pom.xml]: maybe we can use the 2.0 version here?

File: generators/java/templates/pom.xml [hunk 2]
```diff
@@ -4,39 +4,41 @@
...
 	</parent>
 
 	<description><%= integrationName %> connector</description>
-<!-- Properties -->
+	<!-- Properties -->
 	<properties>
-		<service-integration-sdk.version>1.42</service-integration-sdk.version>
+		<service-integration-sdk.version>1.45</service-integration-sdk.version>
```
Comment (nicolas-antomarioni) [inline generators/java/templates/pom.xml]: we can use 1.48 here

**Comment by fedesassone** [comment]
i.e. :
screen shot 2018-03-09 at 11 46 53

**Comment by fedesassone** [comment]
It's not necessary to add the MetricsModule.class .

**Comment by juan-blasco** [comment]
Addressed @nicolas-antomarioni comments

**Comment by fedesassone** [comment]
Metrics maybe for next version .

**Comment by juan-blasco** [comment]
Ready for QA -> @gastontulipani 

@nicolas-antomarioni please add Ignacio Audi to this repo

**Comment by juan-blasco** [comment]
And labels for this project, thanks

**Comment by nicolas-antomarioni** [comment]
Done

**Pull request closed without merging**